aws/ami/windows/windows.pkr.hcl

```diff
@@ -0,0 +1,47 @@
+data "amazon-ami" "windows_root_ami" {
+  filters = {
+    name                = "Windows_Server-2019-English-Tesla-*"
+    root-device-type    = "ebs"
+    virtualization-type = "hvm"
+  }
+  most_recent = true
+  owners      = ["amazon"]
+  region      = "us-east-1"
+}
+
+locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
+
+source "amazon-ebs" "windows_ebs_builder" {
+  ami_name                    = "Windows 2019 GHA CI - ${local.timestamp}"
+  associate_public_ip_address = true
+  communicator                = "winrm"
+  instance_type               = "p3.2xlarge"
+  launch_block_device_mappings {
+    delete_on_termination = true
+    device_name           = "/dev/sda1"
+    volume_size           = 64
+  }
+  source_ami     = "${data.amazon-ami.windows_root_ami.id}"
+  region         = "us-east-1"
+  ami_regions    = ["us-east-1", "us-east-2"]
```

Comment:
This allows us to copy our AMI to both the `us-east-1` and `us-east-2` regions without having to do a reubuild

Reply:
are we going to support multi-regions? If so, other things might need to be replicated to that region as well

Reply:
Currently the canary region for `seemethere/test-repo` is `us-east-2` so that we don't have the potential of reaching capacity for `us-east-1`